src/kusto/azext_kusto/tests/latest/example_steps.py

```diff
@@ -739,12 +752,10 @@ def step_private_endpoint_connection_delete(test, checks=None):
              '--name "{myPrivateEndpointConnection}" '
              '--resource-group "{rg}" ',
              checks=checks)
-    test.cmd('az network private-endpoint delete '
```

Comment:
Could you please tell the reason why we need to remove this command in test?

Reply:
@jsntcy  otherwise tests fail although all pass locally. you can see previous runs
 -
vcr.errors.CannotOverwriteExistingCassetteException: Can't overwrite existing cassette ('/home/vsts/work/1/s/src/kusto/azext_kusto/manual/tests/latest/recordings/test_kusto_Scenario.yaml') in your current record mode ('once'). No match for the request (<Request (GET) [https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/testrg/providers/Microsoft.Network/privateEndpoints/kustoPrivateEndpoint4?api-version=2021-05-01>](https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/testrg/providers/Microsoft.Network/privateEndpoints/kustoPrivateEndpoint4?api-version=2021-05-01%3E)) was found. Found 1 similar requests with 0 different matcher(s) : 1 - (<Request (GET) [https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/testrg/providers/Microsoft.Network/privateEndpoints/kustoPrivateEndpoint4?api-version=2021-05-01>).](https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/testrg/providers/Microsoft.Network/privateEndpoints/kustoPrivateEndpoint4?api-version=2021-05-01%3E).) Matchers succeeded : ['method', 'scheme', 'host', 'port', 'path', '_custom_request_query_matcher'] Matchers failed : -